cfgov/v1/atomic_elements/organisms.py

```diff
@@ -568,7 +568,7 @@ class HTMLBlock(blocks.StructBlock):
         label='Source URL',
         default='',
         required=True,
-        regex=r'^https://.+$',
+        regex=r'^https://s3.amazonaws.com/files.consumerfinance.gov/.+$',
```

Comment:
Can we make the S3 part optional so that we can seamlessly transition to `https://files...` in the future? 

I'm thinking: `^https://(s3.amazonaws.com/)?files.consumerfinance.gov/.+$`